src/shm_ext.c

```diff
@@ -498,6 +498,27 @@ sr_shmext_print(sr_mod_shm_t *mod_shm, sr_shm_t *shm_ext)
     free(items);
 }
 
+static void
+sr_shmext_change_sub_remove_dead(sr_conn_ctx_t *conn, sr_mod_t *shm_mod, sr_datastore_t ds)
```

Comment:
Please include doxygen for all static functions.

Reply:
Sorry, missed that. added now